sklearn/utils/estimator_checks.py

```diff
@@ -396,6 +399,10 @@ def set_checking_parameters(estimator):
     if name == 'OneHotEncoder':
         estimator.set_params(handle_unknown='ignore')
 
+    # set voting='soft' to be able to use predict_proba
+    if name == 'VotingClassifier':
+        estimator.set_params(voting='soft')
```

Comment:
Is this necessary to pass the test or do you just add it so we can test the version with ``predict_proba``? I'd really rather not add anything here, and if we want to test this particular instantiation, we should call ``check_estimator`` on ``VotingClassifier`` with these parameters directly.

This should be necessary for making the tests pass if the ducktyping works correctly.

Reply:
This is necessary otherwise `predict_proba` is not defined and raise an error (leading to some failure in the common tests).

Reply:
#14287 might have fixed this?

Reply:
right